### PR TITLE
Add Julia programming language

### PR DESCRIPTION
**Pull request by cormullion**
I'll be very pleased (and a bit surprised) if I've made no mistakes adding a Julia icon.

### CODE REVIEW

File: images/svg/julia.svg [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<svg
+xmlns="http://www.w3.org/2000/svg"
+aria-label="..." role="img"
```
Comment (edent) [inline images/svg/julia.svg]: You need to add some text in the aria label.

File: images/svg/julia.svg [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+viewBox="0 0 512 512">
+<rect
+width="512" height="512"
+rx="15%"
+fill="#fff"/>
+<circle fill="#389826" cx="256" cy="137" r="83"/>
+<circle fill="#cb3c33" cx="145" cy="329" r="83"/>
```
Comment (edent) [inline images/svg/julia.svg]: You can remove the newlines to save a few bytes.

File: images/svg/julia.svg (hunk 1 above)
Comment (cormullion) [reply images/svg/julia.svg]: Ha, I thought that might be some SVG/XML wizardry... :)

File: images/svg/julia.svg (hunk 2 above)
Comment (cormullion) [reply images/svg/julia.svg]: 🏌️‍♂️

**Pull request merged**